### DISCUSSION TITLE
Advanced Folder Permissions

### DISCUSSION THREAD

**Discussion by coolcat-creations** [Quantum-Manager/tracker#72] [Ideas]
It would be great to have it possible to set up folder permissions to user groups. So that specific user groups have only access to their folders. I am also interested in supporting this feature.

**Comment by b2z** [comment]
Hello.

I think this could help https://www.norrnext.com/docs/joomla-extensions/quantum-manager/solutions#3-override-params-via-profiles

**Comment by dmitriitux** [comment]
Hi, you can use scopes and scopescustom parameter overrides. The scopes parameter is not specified, but it can also be overridden in the same way as scopescustom. The manager does not work directly with some folder. Its entry points are the scopes. Redefine the scopes for the roles you need.
